src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <TestDisabled Condition="'$(IsTestProject)'!='true' Or '$(SkipTests)'=='true' Or ('$(IsPerformanceTestProject)' == 'true' AND '$(Performance)' == 'false')">true</TestDisabled>
-  </PropertyGroup>
 
   <!-- This is the target that copies the test assets to the test output -->
-  <Import Condition="'$(TestDisabled)' != 'true'" Project="$(MSBuildThisFileDirectory)publishtest.targets" />
-  <Import Condition="'$(TestDisabled)' != 'true' AND '$(IsPerformanceTestProject)' == 'true'" Project="$(MSBuildThisFileDirectory)PerfTesting.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)publishtest.targets" />
+  <Import Condition="'$(IsPerformanceTestProject)' == 'true' AND '$(Performance)' != 'false'" Project="$(MSBuildThisFileDirectory)PerfTesting.targets" />
   <UsingTask TaskName="GenerateTestExecutionScripts" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
   <UsingTask TaskName="GetTargetMachineInfo" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
   <UsingTask Condition="'$(BuildingUAPVertical)' == 'true'" TaskName="ExtractResWResourcesFromAssemblies" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
+    <SkipTestRun Condition="'$(IsTestProject)' != 'true' OR ('$(IsPerformanceTestProject)' AND '$(Performance)' == 'false')">true</SkipTestRun>
+    <TestDisabled Condition="'$(SkipTests)'=='true'">true</TestDisabled>
```

Comment:
What is the difference between `TestDisabled` and `SkipTests`?

Reply:
SkipTests => Do not run the tests but still invoke all the targets in tests.targets, i.e. GenerateTestExecutionScripts so that they can be run on helix
TestDisabled => SkipTests + lazy evaluation later for test configs (https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets#L628)